homework01_sergio/vector.h

```diff
@@ -0,0 +1,30 @@
+class Vector {
+public:
+
+    Vector(int x, int y)
+        : mX(x)
+        , mY(y)
+    {}
+
+    int GetX() const {
+        return mX;
+    }
+
+    int GetY() const {
+        return mY;
+    }
+
+    void Add(const Vector& newVector) {
+        mX += newVector.mX;
+        mY += newVector.mY;
+    }
+
+    bool operator==(Vector newVector) {
```

Comment:
Also, you did not do this mistake for `Add`, but here you are copying the Vector